packages/terra-time-input/tests/wdio/time-input-twelve-hour-spec.js

```diff
@@ -126,29 +66,28 @@ Terra.describeViewports('Time Input Twelve Hour', ['medium'], () => {
 
   describe('pressing -', () => {
     it('should roll over to 11 AM if hour and minute is at 12:00 PM', () => {
-      browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/filled-evening');
+      browser.url('/raw/tests/cerner-terra-framework-docs/time-input/twelve-hour/default');
```

Comment:
Why are these URLs changed what was the issue with the existing url?

Reply:
Typed input is not rendering If we use filled input field, that's why i have used default empty field test example

Reply:
So that means the existing functionality is breaking. In previous implementation you could type without clearing the field. Please try to fix.

Reply:
I have verified it, previous implementation is working fine, we can able to edit filled time when we navigate using tab key, but when we click on field using mouse that time only we could not able to edit the filled time.

Reply:
So please look into it if that can be fixed

Reply:
Updated